presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixCaching.java

```diff
@@ -159,6 +175,8 @@ private FileSystem getCachingFileSystem(
                         (dynamicConfig, ignoredContext, ignoredUri) -> {
                             // make sure that dummy cluster manager is used
                             setPrestoClusterManager(dynamicConfig, DummyClusterManager.class.getName());
+                            // ensure new caching FS (with newly initialized BookKeeper) is created for each test case
+                            setCacheKey(dynamicConfig, randomUUID().toString());
```

Comment:
I am not a big fan that you are overriding the the cache key set by `RubixConfigurationInitializer`. I would rather prefer forcing filesystem cache flushing before each test. Is that possible?